projects/mysadna2/src/app/app-routing.module.ts

```diff
@@ -1,7 +1,17 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 
-const routes: Routes = [];
+import { UsersComponent } from './users/users.component';
+import { UserDetailsComponent } from './user-details/user-details.component';
+import { ProjectsComponent } from './projects/projects.component';
+import { ProjectDetailsComponent } from './project-details/project-details.component';
+
+const routes: Routes = [
+  {path:'users', component: UsersComponent}, 
+  {path:'user-details', component: UserDetailsComponent},
+  {path:'projects', component: ProjectsComponent},
+  {path:'project-details', component: ProjectDetailsComponent},
```

Comment:
Also, please try run `ng lint`
You'll see several errors there. To fix it run: `ng lint --fix` (or `ng lint -fix), it depends on `ng` version)
Some errors can be fixed by manual way only.
I'll tell you about linting and tslint rules soon.